autobot/api.py

```diff
@@ -8,77 +8,73 @@
 
 """Autobot API."""
 
+import copy
+
 import yaml
 
 from autobot.config_loader import Config
 from autobot.github import GitHubAPI
 
 
 class BotAPI:
-    """Generates report."""
+    """Bot's API."""
 
-    @classmethod
-    def __init__(cls, config: Config):
+    def __init__(self, config: Config):
         """Bot initialization."""
-        cls.config = config.config
-        cls.report = GitHubAPI(
-            cls.config["AUTOBOT_OWNER"], cls.config["AUTOBOT_GH_TOKEN"]
-        ).report(cls.load_repositories())
+        self.config = config
 
     @classmethod
-    def generate_report(cls, maintainer: str) -> dict:
-        """Generates a report for a maintainer."""
-        res = cls.report[0]["repos"]
-        print(res)
-        print(
-            "---------------------------------------------------------------------------------"
-        )
-        return res
+    def md_report(cls, report):
+        """Returns the report in markdown format."""
+        lines = []
+        for repo in report.keys():
```

Comment:
```suggestion
        for repo, repo_report in report.items():
```

That's a bit more idiomatic, in cases where you want both the key and the value of a dictionary